### PR TITLE
Deleted Scooping Requirements

### PR DESCRIPTION
**Pull request by NsubugaAnold**
#### Summary of Changes (What does this PR do?)
- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @NsubugaAnold 
Looks good to me

File: docs/hardware/monitor.md [hunk 1]
```diff
@@ -98,21 +98,6 @@ mounting. Before installation, take note of the following pre-installation preca
 - Mount the monitor at normal human breathing height (about 1.5-2m above the
   ground)
```
Comment (Baalmart) [inline docs/hardware/monitor.md:100]: What is the motivation for removing these scoping requirements? @NsubugaAnold 

cc: @123MwanjeMike

Comment (123MwanjeMike) [reply docs/hardware/monitor.md:100]: That is the current state of the manual as @NsubugaAnold  can confirm @Baalmart

**Review by Codebmk — APPROVED** [review]

**Review by Baalmart — APPROVED** [review]
LGTM

File: docs/hardware/monitor.md (hunk 1 above)
Comment (Baalmart) [reply docs/hardware/monitor.md:100]: Alright, thanks @123MwanjeMike

**Pull request merged**